src/Components/CloudAccounts/CloudAccountsUtils.tsx

```diff
@@ -0,0 +1,22 @@
+import {
+  CloudProvider,
+  CloudProviderId,
+  CloudProviderShortName,
+} from './CloudAccountProviders';
+
+export const CloudProviderLabelMap: Record<
+  CloudProviderShortName,
+  CloudProvider
+> = {
+  AWS: 'AWS',
+  GCE: 'Google Cloud',
+  MSAZ: 'Azure',
+};
+
+export const providerToApiParam: Record<CloudProvider, CloudProviderId> = {
+  AWS: 'aws',
+  Azure: 'azure',
+  'Google Cloud': 'gcp',
+} as const;
+
+export type ProviderKey = keyof typeof CloudProviderLabelMap;
```

Comment:
Nitpick: I'm conflicted about some of these type definitions. Lets chat about them